### ISSUE TITLE
opensubtitles.org API returns stub subtitle for all searches as it is no longer functional

### ISSUE THREAD

**Issue by Frank071** [pymedusa/Medusa#11643]
**Describe the bug**
The opensubtitles.org API no longer functions for non VIP members, but 'no longer functions' is implemented as 'responds with a one line subtitle to every request'. This means that applications like Medusa thinks it has the subtitle and downloads it.

**To Reproduce**
Check any subtitle downloaded via the opensubtitles API after Jan 1st, 2024

**Expected behavior**
To avoid misinterpretation it is better to remove the opensubtitles provider. Users with VIP access can activate the opensubtitlesvip provider and it will continue to work (for now). Those without VIP access will no longer get these stub subtitles.

**Additional context**
Supporting the new API would be a nice update for Medusa. Information can be found [here](https://opensubtitles.stoplight.io/docs/opensubtitles-api/).

**Comment by BenjV** [comment]
I would(and could) be willing to create the code for the new Opensubtitle API.
But a Medusa developer has to create a API key for medusa on Opensubtitle site because that is needed for the new Opensubtitles API.

**Comment by Frank071** [comment]
I don't think that will happen as the free use of the API is pretty limited. If you register as user of opensubtitles, you van create an API key yourself. Wouldn't it be a better solution to make this key a configurable option? In addition to to the current user/pass combination (it looks as if the use of the API key when logged in is less limited).

(But it might be that I completely misunderstood the new API scheme)

**Comment by 007revad** [comment]
Is there any update on this?

**Comment by BenjV** [comment]
No response from any of tehe developers!

**Comment by gazmeist** [comment]
Yup turned off opensubtitles for exactly this reason - too many useless 1kb files. Only allows you 10 files a day
